components/eam/src/physics/cam/physpkg.F90

```diff
@@ -274,6 +275,7 @@ subroutine phys_register
        call prescribed_volcaero_register()
        call prescribed_ozone_register()
        call prescribed_aero_register()
+       call read_spa_data_register()
```

Comment:
call only if SPA is active

Reply:
I think CCN3 is only being defined in pbuf if SPA is active, but is being grabbed all the time. Probably need `if SPA` conditional around these other calls too?